sequencer.c

```diff
@@ -1425,7 +1447,7 @@ static int try_to_commit(struct repository *r,
 	if (parse_head(r, &current_head))
 		return -1;
 	if (!(flags & AMEND_MSG) && opts->committer_date_is_author_date &&
-	    setenv_committer_date_to_author_date())
+	    !opts->ignore_date && setenv_committer_date_to_author_date())
```

Comment:
What if someone passes both --committer-date-is-author-date and --ignore-date?  Do they expect author date and committer date to get swapped?  Right now, this appears to make --ignore-date override --committer-date-is-author-date -- perhaps we should error out instead?  (Or maybe this is fine as-is because git-am also surprisingly just makes --ignore-date silently override --committer-date-is-author-date, without documenting it, so you've at least matched the behavior elsewhere...)

Not sure you need to do anything here, especially since you do match git-am behavior, but it might be worth pointing out in your cover letter.